packages/driver/cypress/integration/e2e/multi-domain/multi_domain_spec.ts

```diff
@@ -275,4 +275,20 @@ describe('domain validation', { experimentalSessionSupport: true }, () => {
     .invoke('text')
     .should('equal', 'Welcome TJohnson')
   })
+
+  it('creates a spec bridge for https://idp.com:3502', () => {
```

Comment:
Should these tests be in `multi_domain_validation_specs` where we are validating the src on the spec bridges?

Reply:
🤷‍♂️  sure, I thought that one was more about validating command params, but i can move this whole describe block over there.